retrofit-converters/moshi/src/main/java/retrofit2/converter/moshi/MoshiConverterFactory.java

```diff
@@ -52,37 +52,49 @@ public static MoshiConverterFactory create() {
   @SuppressWarnings("ConstantConditions") // Guarding public API nullability.
   public static MoshiConverterFactory create(Moshi moshi) {
     if (moshi == null) throw new NullPointerException("moshi == null");
-    return new MoshiConverterFactory(moshi, false, false, false);
+    return new MoshiConverterFactory(moshi, false, false, false, false);
   }
 
   private final Moshi moshi;
   private final boolean lenient;
   private final boolean failOnUnknown;
   private final boolean serializeNulls;
+  private final boolean requireFullResponseConsumption;
```

Comment:
I'm not sure we need a knob for this...

Reply:
yeah. seems unlikely to break any current users.